docs/domains/Domain.json

```diff
@@ -117,6 +117,11 @@
           "description": "Customization affecting Kubernetes Service generated for this WebLogic cluster.",
           "$ref": "#/definitions/KubernetesResource"
         },
+        "maxConcurrentShutdown": {
+          "description": "The maximum WebLogic Server instances that will shutdown in parallel for this cluster when it is being partially shutdown by lowering its replica count.A value of 0 means there is no limit. Defaults to `spec.maxClusterConcurrentShutdown` (which in turn defaults to 1).",
```

Comment:
I agree with all the edits you proposed.
The maximum WebLogic Server instances -> The maximum number of WebLogic Server instances
shutdown  -> shut down (globally, shut down (v), shutdown (n) )
Space needed after count. 
which in turn defaults to 1 -> which defaults to 1